data/json/recipes/recipe_electronics.json

```diff
@@ -47,6 +47,25 @@
       [ [ "scrap", 1 ], [ "steel_chunk", 1 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "reading_light",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_LIGHTING",
+    "skill_used": "electronics",
+    "difficulty": 2,
+    "time": "20 m",
+    "reversible": true,
+    "decomp_learn": 2,
+    "book_learn": [ [ "manual_electronics", 3 ], [ "advanced_electronics", 3 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [
```

Comment:
This needs something representing a lightbulb or LED, e_scrap would be fine (meaning LEDs)

Reply:
Should the core recipe of atomic reading light get one as a component too, then? I copied most of the recipe from there, but took out the betavoltaic cell (so you get to provide power source separately).

Reply:
Yes.